test/suites/container_devices_nic_bridged.sh

```diff
@@ -36,6 +36,17 @@ test_container_devices_nic_bridged() {
   # Record how many nics we started with.
   startNicCount=$(find /sys/class/net | wc -l)
 
+  # Test device renaming without "eth0" device applied by profile.
+  lxc profile copy default "${ctName}"
+  lxc profile device remove "${ctName}" eth0
+  lxc init testimage "${ctName}" -p "${ctName}"
```

Comment:
If the instance can be offline it'd be best to go with:

```suggestion
  lxc init --empty "${ctName}" -d "${SMALL_ROOT_DISK}" -p "${ctName}"
```

but if it needs to be running:

```suggestion
  lxc launch testimage "${ctName}" -d "${SMALL_ROOT_DISK}" -p "${ctName}"
```

FYI, I intend to do a mass change to add those `-d "${SMALL_ROOT_DISK}"` bits but I have yet to get around to doing it :/

Reply:
> FYI, I intend to do a mass change to add those `-d "${SMALL_ROOT_DISK}"` bits but I have yet to get around to doing it :/

Do you think a `gopatch` would help here, or are there enough exceptions where `SMALL_ROOT_DISK` isn’t applicable that we’d need to review each `lxc init/launch` manually?

Reply:
For now I've added `-d "${SMALL_ROOT_DISK}"` where applicable in the container device NIC bridge tests.

Reply:
I've ran into regressions when applying it massively without much consideration. That's why I put the PR back on the back burner :/

Thanks for taking care of that script!